apps/scoutgame/app/(general)/info/repositories/page.tsx

```diff
@@ -6,6 +6,8 @@ export const metadata: Metadata = {
   title: 'Repositories'
 };
 
+export const dynamic = 'force-dynamic';
```

Comment:
hoping to fix this build error:
![image](https://github.com/user-attachments/assets/128c9700-0dad-4ddf-bd35-be3b52acc082)